src/main/resources/templates/dataresource.html

```diff
@@ -0,0 +1,184 @@
+						<th width="15%">Release Date</th>
+						<th width="30%">Action</th>
+					</tr>
+					<tr>
+						<td>1</td>
+						<td><a href="#" data-toggle="modal" data-target="#myModal">ITDK 2015-08</a></td>
+						<div class="container">
+							<div id="myModal" class="modal fade admin-modal" role="dialog">
+								<div class="modal-dialog">
+									<!--Modal Content-->
+									<div class="modal-content">
+										<div class="modal-header">
+											<button type="button" class="close" data-dismiss="modal">&times;</button>
+											<h4 class="modal-title">ITDK 2015-08 </h4>
+										</div>
+										<div class="modal-body">
+											<p><b>Details</b>
+											<p>This dataset enables the study of the topology of the IPv4 and IPv6 Internet at the router-level with inferences for assignments of routers to Autonomous Systems (AS). The August 2015 release of the ITDK includes two related IPv4 router-level topologies; an IPv6 router-level topology; assignments of routers to ASes; geographic locations of each router; and Domain Name Service (DNS) lookups of all observed IP addresses.</p>
+											</p>
+											<p>
+												<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+												<button type="button" class="btn btn-standard">Save</button>
+											</p>
+
+										</div>
+										<div class="modal-footer">
+											<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
+						<td>Study of the topology of the IPv4 and IPv6 Internet</td>
+						<td>2015/08</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>2</td>
+						<td>DNS AMPL DDOS DEC2015 </td>
+						<td>DNS Server Spoofed Request Amplification DDoS</td>
+						<td>2015/07</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+							<td>2</td>
+							<td>SSDP DDoS attack 2015</td>
+							<td>Simple Service Discovery Protocol attack</td>
+						    <td>2015/12</td>
+							<td>
+								<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+								<button type="button" class="btn btn-standard">Save</button>
+					</td>
+					</tr>
+
+
+					</table>
+					<hr/>
+					<div class="fa fa-share" ><strong> &nbsp;&nbsp;Link to <a href="https://www.impactcybertrust.org/"> IMPACT Data</a></strong></div>
+				</div>
+				<!--<p>Some content.</p>-->
+			</div>
+			<div id="bigdata" class="tab-pane fade">
+				<!--<h3>Menu 1</h3>-->
+				<!--<p>Some content in menu 1.</p>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)